### DISCUSSION TITLE
Autoplay

### DISCUSSION THREAD

**Discussion by codinPenguin** [pystardust/ani-cli#856] [Ideas] [answered]
What are the chances of they devs of ani-cli adding an auto play feature.

**Comment by justchokingaround** [comment, marked as answer]
i'd say around 100% 

||since it already exists||

**Reply by justchokingaround** [reply]
<img width="664" alt="image" src="https://user-images.githubusercontent.com/44473782/182162866-8d9beb88-9a72-4399-ad6e-10e2056c6b61.png">

ex: auto play episodes from 1 to 4

**Reply by codinPenguin** [reply]
oop thx so much XD
